strimzi-kafka-operator/launch_java.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -e
+set +x
+
+# expand gc options based upon java version
+function get_gc_opts {
+  if [ "${STRIMZI_GC_LOG_ENABLED}" == "true" ]; then
+    echo "-Xlog:gc*:stdout:time -XX:NativeMemoryTracking=summary"
+  else
+    # no gc options
+    echo ""
+  fi
+}
+
+export MALLOC_ARENA_MAX=2
+
+# Make sure that we use /dev/urandom
+JAVA_OPTS="${JAVA_OPTS} -Djava.security.egd=file:/dev/./urandom"
+
+# Enable GC logging for memory tracking
+JAVA_OPTS="${JAVA_OPTS} $(get_gc_opts)"
+
+# Exit when we run out of heap memory
+JAVA_OPTS="${JAVA_OPTS} -XX:+ExitOnOutOfMemoryError"
+
+# Default memory options used when the user didn't configured any of these options, we set the defaults
+if [[ "$JAVA_OPTS" != *"MinRAMPercentage"* && "$JAVA_OPTS" != *"MaxRAMPercentage"* ]]; then
+  JAVA_OPTS="${JAVA_OPTS} -XX:MinRAMPercentage=15 -XX:MaxRAMPercentage=20"
+fi
+
+# Disable FIPS if needed
+if [ "$FIPS_MODE" = "disabled" ]; then
+    JAVA_OPTS="${JAVA_OPTS} -Dcom.redhat.fips=false"
+fi
```

Comment:
Do we need this? seems unrelated to this type of package?

Reply:
I copied it directly from here: https://github.com/strimzi/strimzi-kafka-operator/blob/main/docker-images/operator/scripts/launch_java.sh

Reply:
do you want me to patch this line to remove it?